### PR TITLE
Icon things

### LINKED ISSUE #141: All the icon decisions
**Icons for plain string requirements:**
assignment_late for requirements with no manual progress set
assignment for requirements with a manual progress set (has a key in the progressOverrides dict)
assignment_turned_in for completed requirements (also make it green)

**Icons for regular requirements:**
Check (done) for completed
Dash (remove) for uncompleted

**Include icons in click event to open card**

**All requirements with progress bar**
Progress bar only green when full, otherwise yellow

**Info icons**
Only show on hover

**Close icons**
Courses on left
Everything else on top right

**Percentages show on hover**

### PR DESCRIPTION
**Pull request by georgiashay**
Closes #141 

Changes from description in #141:
-Added an orange color in addition to yellow on the progress bar (<15% complete)
-Can't access the dropdown icons, so instead of the tooltips, the percentage now displays next to the info icon (open to suggestions on better ways of displaying this, but I think it looks nice)

Also an idea that came up during this process was to make the icon buttons black then fade to grey when they appear to draw attention to them.  Could make an issue for backburner if this is a good idea.

### CODE REVIEW

File: src/components/Requirement.vue [hunk 1]
```diff
@@ -3,36 +3,44 @@
...
-      <span v-if = "'title' in req">| </span>
-      <span style = "text-transform: cursive">{{ req.req }}</span>
-    </span>
-    <span v-if = "req.max === 0 && leaf" style = "font-style:italic">
-       (optional)
-    </span>
-    <div :class = "percentage_bar" :style = "percentage"></div>
+  <v-layout row>
+    <v-flex>
+      <div>
```
Comment (npfoss) [inline src/components/Requirement.vue]: do we need this div? I tested removing it and it didn't seem to break anything, but I assume it's here for a reason?

File: src/components/Requirement.vue [hunk 2]
```diff
@@ -43,8 +51,8 @@ export default {
   data: function() {
     return {
       open: [],
-      viewDialog: false,
-      showDelete: false
+      hoveringOver: false,
+      iconHover: false,
```
Comment (npfoss) [inline src/components/Requirement.vue]: I've noticed that you started adopting my comma convention recently. It's nice for diffs!

File: src/components/Requirement.vue (hunk 1 above)
Comment (georgiashay) [reply src/components/Requirement.vue]: Yeah, I can't remember why that div was there but it's obviously no longer necessary.

**Comment by npfoss** [comment]
(testing our first squash and merge)

**Pull request merged**